### DISCUSSION TITLE
Documentation Improvment: Please add guide for upgrading using docker-compose

### DISCUSSION THREAD

**Discussion by ptmrio** [umami-software/umami#621] [General]
Could you please provide a upgrade guide for the docker-compose install variant in the documentation?

Afaik is should simply be `docker-compose pull`. Yet I'm pretty new to docker. While this updates the image, what happens to db schema upgrades? A official documentation on this would be pretty neat, as docker-compose is mentioned as a way of installation.

**Comment by k3tan172** [comment]
Wondering if there's been any updates to this? I'm running umami in docker. If I run `docker-compose pull`, will this upgrade to the latest version? Are there any further instructions?

**Comment by tycrek** [comment]
@ptmrio @k3tan172 I think I figured it out, at least these worked for me:

```bash
docker-compose pull
docker-compose up --force-recreate
```

I've opened #1029 to hopefully add these to the README for future users.

**Reply by ptmrio** [reply]
Thanks. Currently I'm simply updating by `docker-compose pull` & `docker-compose up`.

`--force-recreate` does not seem necessary atm to me. Admin somewhere wrote, that the database structure did not change till now, so no need to upgrade the schema (atm!). Let's see what the future brings :)
